### PR TITLE
Fix filename completions

### PR DESCRIPTION
**Pull request by bootandy**
I previously accidentally reverted a commit that @wickles made to fix the completion.

https://github.com/bootandy/dust/issues/359

### CODE REVIEW

**Comment by wickles** [comment]
Maybe I'm wrong but I think there needs to be an adjustment in `build.rs` or `src/cli.rs`, or else `clap` will just revert this again with the next build.

File: completions/_dust [hunk 1]
```diff
@@ -71,6 +71,7 @@ _dust() {
 '-V[Print version]' \
 '--version[Print version]' \
 '*::params:' \
+'*:inputs:_files' \
```
Comment (wickles) [inline completions/_dust:74]: This will not work. Remove the new line and replace the existing line with
```
'*::params:_files' \
```

**Review by wickles — CHANGES_REQUESTED** [review]

File: completions/_dust (hunk 1 above)
Comment (bootandy) [reply completions/_dust]: Happy to make this change. 

When I test tab completion it seems to work for me. 

How would you recommend that I test this? & How do you know this is the right thing to do ? 

Thanks

**Comment by bootandy** [comment]
>Maybe I'm wrong but I think there needs to be an adjustment to the completion generation in build.rs and src/cli.rs, or else clap will just revert this again with the next build.

You are right, going to rethink this.

**Comment by bootandy** [comment]
There's a lot of auto complete issues raised in clap-complete. https://github.com/clap-rs/clap/issues?page=2&q=is%3Aissue+is%3Aopen+completion

A (rather hacky) solution could be to add a comment saying 'do not commit this change' if clap rebuilds it. Although, ideally we want to fix the root problem which is either in clap-complete or my use of clap.

File: completions/_dust (hunk 1 above)
Comment (wickles) [reply completions/_dust]: Just edit the existing file in your fpath and then start a new zsh session. When I try with both `*:...` lines added:
```
❯ dust <tab>
_arguments:comparguments:327: doubled rest argument definition: *:inputs:_files
_arguments:comparguments:327: doubled rest argument definition: *:inputs:_files
❯ dust 
```
compared with the one line as in my suggestion:
```
❯ dust <tab>
-- params --
Desktop/  Documents/ ...
```

**Pull request closed without merging**